components/button/src/button/button.js

```diff
@@ -37,6 +37,14 @@ export const Button = ({
         }
     }, [initialFocus, ref.current])
 
+    const { 'aria-label': ariaLabel, title } = otherProps
```

Comment:
not part of this PR - but I think `ariaLabel` in the previous tests should be updated to `aria-label` like in [this test](https://github.com/dhis2/ui/pull/1451/files#diff-2fdc96d318bd3f7fe855ac745e982336111185c7fd55c50a78151bd533d10c12R30)

Reply:
yeah we should make sure these are `aria-label` (not camelcased). This applies to your previous PR as well @d-rita  - let's update them to make them consistent `aria-*` 

```
In React, all DOM properties and attributes (including event handlers) should be camelCased. For example, the HTML attribute tabindex corresponds to the attribute tabIndex in React. The exception is aria-* and data-* attributes, which should be lowercased. For example, you can keep aria-label as aria-label.

```
https://legacy.reactjs.org/docs/dom-elements.html